distributor-api-v1/operations.md

```diff
@@ -1277,3 +1279,131 @@ In case of an error caused by insufficient availability \(which might have decre
 - `Charged` - Finite state. Payment has been successfully charged.
 - `Canceled` - Finite state. Payment has been canceled, and it has not been charged.
 - `Failed` - Finite state. Payment has not been charged.
+
+## Get Payment Cards <a id="get-payment-cards"></a>
```

Comment:
Why is each word in first capital format?

Reply:
Consistence for now:
![image](https://user-images.githubusercontent.com/6573164/116701164-e0cc3f00-a9c7-11eb-8fd2-3a27baa43c9c.png)

But definetelly a good point also for the next PR which will be consolidation